Cargo.toml

```diff
@@ -55,6 +55,7 @@ disable-linker-script = []
 
 # STM32L0 subfamilies
 # (Warning: Some peripherals, e.g. GPIO, don't follow this subfamily grouping.)
+stm32l0x0 = ["stm32l0/stm32l0x0"]
```

Comment:
Is that enough, or do we need to add more features flags (thinking about the ones @dbrgn auto-generated from CubeMX).

Reply:
probably yes. I only have a l0x0RBT to test against, which is a class 5. I keep finding more things to fix; basically it's similar to the features of L020 class 5 with a lot of stuff cut out.